go/parquet/file/row_group_reader.go

```diff
@@ -31,7 +30,7 @@ const (
 
 // RowGroupReader is the primary interface for reading a single row group
 type RowGroupReader struct {
-	r             ipc.ReadAtSeeker
+	r             parquet.ReaderAtSeeker
```

Comment:
This is more of a typo correction?

Reply:
it didn't need to point at the `ipc.ReadAtSeeker` (which has the extra restriction of the `Read` method. it should be using `parquet.ReaderAtSeeker` which has fewer requirements.